waku/wms.md

```diff
@@ -45,13 +45,25 @@ limit  = 4OCTET
 ; array of a cursor returned from the previous request (optional)
 cursor = 32OCTET
 
-payload = "[" lower upper bloom limit [ cursor ] "]"
+; List of topics interested in
+topic-interest   = "[" *1000topic "]"
+
+; 4 bytes of arbitrary data
+topic = 4OCTET
+
+payload_alt1 = "[" lower upper bloom limit [ cursor ] "]"
```

Comment:
can we name this something better, `payload_without_topic` or something?

Reply:
done